crates/api/src/local_user/save_settings.rs

```diff
@@ -90,9 +90,11 @@ pub async fn save_user_settings(
     ..Default::default()
   };
 
+  // Ignore errors, because 'no fields updated' will return an error.
+  // https://github.com/LemmyNet/lemmy/issues/4076
   Person::update(&mut context.pool(), person_id, &person_form)
     .await
-    .with_lemmy_type(LemmyErrorType::UserAlreadyExists)?;
+    .ok();
```

Comment:
The error comes from `.get_result()`. Instead of this workaround you can simply change the query to `execute()` which also means that it doesnt return unnecessary data.

Reply:
Don't think we should do that, bc it does need to be used elsewhere, and use its result.

Reply:
Hmm too bad. We could consider changing update to return `Result<(), Error>` in the future.